api/agent_access_tokens.go

```diff
@@ -169,3 +169,71 @@ type AgentAccessTokenRequest struct {
 	TokenAlias string                 `json:"tokenAlias,omitempty"`
 	Props      *AgentAccessTokenProps `json:"props,omitempty"`
 }
+
+func (svc *AgentAccessTokensService) Search(response interface{}, filters SearchFilter) error {
```

Comment:
Why is this new function inside the `AgentAccessTokensService`??? This is really wrong!

Every API v2 schema should be its own service 😓 - I will have to refactor this too.